packages/kubernetes-1.23/Cargo.toml

```diff
@@ -17,6 +17,31 @@ package-name = "kubernetes-1.23"
 url = "https://distro.eks.amazonaws.com/kubernetes-1-23/releases/28/artifacts/kubernetes/v1.23.17/kubernetes-src.tar.gz"
 sha512 = "31fd414bebe6ec71682c65256373a99467f558340739739df3f332d6bf1cfef2a2c0b39b337f3f0ed56ae72a4ae083f4fdbe6ad8e964b86c2c9e144aa1f748ab"
 
+[[package.metadata.build-package.external-files]]
+path = "0026-EKS-PATCH-Cherry-pick-119832-Fix-the-problem-Pod.patch.bz2"
+url = "https://github.com/aws/eks-distro/blob/168d252ab1f465d3621a6e9b2251fb72b250c49f/projects/kubernetes/kubernetes/1-23/patches/0026-EKS-PATCH-Cherry-pick-119832-Fix-the-problem-Pod-ter.patch"
+sha512 = "0406fad037a41750310bcc7e75dceaa65cb6d9ffd8e324541068d25074386ff5fbfdb3e6b4d429704e692a39624377483ecd8ed7fcc16d54ba68d81c97d5d270"
+
+[[package.metadata.build-package.external-files]]
+path = "0027-EKS-PATCH-Prevent-rapid-reset-http2-DOS-on-API-serve.patch.bz2"
+url = "https://raw.githubusercontent.com/aws/eks-distro/168d252ab1f465d3621a6e9b2251fb72b250c49f/projects/kubernetes/kubernetes/1-23/patches/0027-EKS-PATCH-Prevent-rapid-reset-http2-DOS-on-API-serve.patch"
+sha512 = "d36a581ec2b4622da52d25e942fc063c7fcccbba08ee3c9d0dd52366ca6cd80300ced9c807ef786544b83e7d1dddcf88e24b478a4a4f32e864e9e8edaea8940d"
```

Comment:
Oh, it looks like the files you've uploaded aren't actually `.bz2` files. That's OK, but if they're not compressed then they should just have the `.patch` extension. That might be preferable because then it's easier to validate that they match the source.